src/libs/logger.js

```diff
@@ -76,14 +77,14 @@ export const logger = winston.createLogger(options);
  *
  * @param {String} port The port the server is running on
  */
-export const started = (port) => {
-  console.log(`${chalk.cyan('Secure Sign')} API started ${chalk.green('✓')}`);
+export const started = (port, appName) => {
+  console.log(`${chalk.cyan(appName || 'Secure Sign')} API started ${chalk.green('✓')}`);
```

Comment:
I think we can change this to be `${apName} API Started` or just 'API Started'. No need to keep the text Secure Sign in there. Want make the change?

Reply:
Sure. Do you want to keep the coloring? i.e. `${chalk.cyan(...)` or wanna get rid of it? I kinda like it...